app/elements/candidates/invite-candidates.html

```diff
@@ -78,14 +74,13 @@
 				}
 			},
 			_addCandidate: function(){
-				if (this.challenge){
-					this.push('candidates', {'Challenge': this.challenge.Key});
+				if (this.challenges.length == 1){
+					this.push('candidates', {'Challenge': this.challenges[0].Key});
```

Comment:
when you open the dialog by default the select has 1 challenge. It is not set in  the candidate object.